build-system/common/ci.js

```diff
@@ -205,6 +205,15 @@ function circleciPrMergeCommit() {
   return isCircleci ? env('CIRCLECI_MERGE_COMMIT') : '';
 }
 
+/**
+ * Returns an identifier that is unique to each CircleCI job. This is different
+ * from the workflow ID, which is common across all jobs in a workflow.
+ * @return {string}
+ */
+function circleciBuildNumber() {
+  return isCircleci ? env('CIRCLE_BUILD_NUM') : '';
```

Comment:
Had to double-check that `CIRCLE_BUILD_NUM` is unique per job, cause the name isn't clear.
([it is](https://circleci.com/docs/2.0/env-vars/))

Reply:
Yep, for posterity, it is this number, which is different for each job.

<img width="1275" alt="Screen Shot 2021-04-07 at 3 48 50 PM" src="https://user-images.githubusercontent.com/26553114/113925270-d4820700-97b8-11eb-902a-930562aec821.png">